config/config.exs

```diff
@@ -9,6 +9,9 @@ config :plug, :types, %{
   "application/vnd.api+json" => ["json-api"]
 }
 
+# Use Jason for JSON parsing in Phoenix
```

Comment:
I think this comment is from the upgrade guide. 
We are still using `Poison` and then we are moving to `Jason`, right?
```
config :phoenix, :json_library, Jason
```

Reply:
Phoenix can use either Poison or Jason from its own parse/dump JSON (be it internal or implicit). Since Phoenix now recommends Jason, and it's supposed to be faster, let's just use it already.

But we still use Poison explicitly to [parse/dump JSON in Surveda](https://github.com/instedd/surveda/search?q=Poison) and these are the cases where we could migrate to use Jason explicitly.